mooclet_engine/engine/views.py

```diff
@@ -220,18 +220,34 @@ def post(self, request):
         if not all(key in req for key in ("learner", "mooclet", "policy")):
             return Response({"error": "invalid request"}, status=500)
 
-        mooclet = Mooclet.objects.get(pk=req["mooclet"])
-        learner = Learner.objects.get(name=req["learner"])
-        policy = Policy.objects.get(pk=req["policy"])
+        try:
```

Comment:
can we make these 4 try except block into one block?

Reply:
The goal here is to find out which part goes wrong and let user know. Is there a better way doing it?